charts/sscs-evidence-share/values.preview.template.yaml

```diff
@@ -4,22 +4,15 @@ java:
   aadIdentityName:
   environment:
     SEND_LETTER_SERVICE_ENABLED: "true"
+    SEND_LETTER_SERVICE_BASEURL: "http://rpe-send-letter-service-aat.service.core-compute-aat.internal"
+    IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-aat.service.core-compute-aat.internal"
+    CORE_CASE_DATA_API_URL: "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
     SUBSCRIPTION_NAME: "sscs-evidenceshare-subscription-aat"
     TOPIC_NAME: "sscs-evidenceshare-topic-aat"
     AMQP_HOST: "sscs-servicebus-aat.servicebus.windows.net"
     AMQP_USERNAME: "SendAndListenSharedAccessKey"
-    PDF_SERVICE_BASE_URL: ${PDF_SERVICE_BASE_URL}rs/render
```

Comment:
ah, this was what made it work on preview...

Reply:
Removing it from bootstrap.yaml really